sklearn/decomposition/_incremental_pca.py

```diff
@@ -418,3 +418,8 @@ def transform(self, X):
             return np.vstack(output)
         else:
             return super().transform(X)
+
+    def __sklearn_tags__(self):
+        tags = super().__sklearn_tags__()
+        tags.input_tags.sparse = True
```

Comment:
This one is really funky: `fit` accepts sparse but not `partial_fit`. So I assume that we can be lenient because we cannot use `partial_fit` in meta-estimator or pipeline.

Reply:
This could be good to add a small comment to acknowledge this specificity.

Reply:
done in 705c414